test/reporters/fixtures/indicator-position.test.ts

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable no-multiple-empty-lines */
```

Comment:
I think this file should be javascript because esbuild truncates all empty lines.